netbout-web/src/main/java/com/netbout/rest/bout/XeAttachment.java

```diff
@@ -93,7 +93,8 @@ private static XeSource make(final Request req, final Bout bout,
                     .add("name").set(atmt.name()).up()
                     .add("ctype").set(atmt.ctype()).up()
                     .add("etag").set(atmt.etag()).up()
-                    .add("unseen").set(Boolean.toString(atmt.unseen()))
+                    .add("unseen").set(Boolean.toString(atmt.unseen())).up()
+                    .add("author").set(atmt.author())
```

Comment:
@mkordas Same thing, the puzzle I added. I can't fetch dates with `DyAttachment` yet, so we cannot send it here. It craches, because there is no row for `date` in remote database.